pull_request_template.md

```diff
@@ -0,0 +1,118 @@
+## PR Writing Checklist
+
+If your PR involves adding or updating writing content,
+please check the following items before submitting your PR for review:
+
+- [ ] **Avoid [inline links].
```

Comment:
OK, makes sense. If we want to use both inline links and lists of reference link definitions, should we include a one-liner for guidance on when to use each option?

Reply:
> We do generally use this rule for a codelab, where we want to minimize distractions, but not for other docs.

When we talk about minimizing distractions in codelabs, I'm interpreting this to mean that we minimize distractions **for the codelab participant**. Since reference link / inline link distinction is completely hidden from the user, I'm not sure why this applies here. LMK if I've missed something.

Reply:
Oh, wait!! I think I misunderstood your original point. Completely. I thought you were saying, don't link on text in prose. Put links at the bottom of the page. But now I realize you were saying to *define* the links at the bottom of the page.  I agree with this. But you need to rephrase to make it clearer. (It would have been easier if you had first created this list in a Google doc and reached consensus before making a PR. In fact, it's not too late to do that.  :) )

Reply:
Sure, I'll throw this in a doc!